docs/get-started/using-with-map.md

```diff
@@ -15,96 +17,13 @@ There are two types of integration between deck.gl and a base map solution:
-
-Open source tile schemas include:
-
-- [TileZen schema](https://tilezen.readthedocs.io/en/latest/layers/)
-- [OpenMapTiles schema ](https://openmaptiles.org/schema/)
-
-Some useful resources for creating your own map service:
-
-- [Mapbox Vector Tile Spec](https://www.mapbox.com/developers/vector-tiles/)
-- [Open source tools](https://github.com/mapbox/awesome-vector-tiles)
-- [Maputnik Style editor](https://maputnik.github.io)
-
-### Compatibility with Mapbox GL JS forks
-
-As of v2.0, Mapbox GL JS [went proprietary](https://github.com/mapbox/mapbox-gl-js/blob/main/CHANGELOG.md#200) and requires a Mapbox account to use even if you don't load tiles from the Mapbox data service. Community forks of the v1 code base such as [MapLibre GL JS](https://maplibre.org) can generally be used as a drop-in replacement of mapbox-gl. If you are using react-map-gl, follow [these instructions](http://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork).
-
-If the forked libraries and Mapbox API diverge in the future, compatibility issues may arise. deck.gl intends to support open source efforts wherever reasonable. Please report any issue on GitHub.
-
-
-## Using deck.gl with Google Maps
-
-Starting with v7.0, deck.gl has support for Google Maps with the [@deck.gl/google-maps](/docs/api-reference/google-maps/overview.md) module. It allows you to construct a Deck instance as a custom Google Maps [OverlayView](https://developers.google.com/maps/documentation/javascript/reference/#OverlayView).
-
-![deck.gl as a Google Maps overlay](https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/google-maps.jpg)
-
-The Deck canvas can only be used as a overlay on top of Google Maps, see [pure JS example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/google-maps). Tilting is not supported due to Google Maps API restrictions. See module documentation page for a full list of features.
-
-Note that to run the examples, you need a [Google Maps API key](https://developers.google.com/maps/documentation/javascript/get-api-key).
-
-
-## Using deck.gl with ArcGIS
-
-Starting with v8.1, deck.gl has support for ArcGIS with the [@deck.gl/arcgis](/docs/api-reference/arcgis/overview.md) module.
-
-![deck.gl as a ArcGIS map layer](https://raw.github.com/visgl/deck.gl-data/master/images/whats-new/arcgis.jpg)
-
-2D integration with `MapView` is supported by the [DeckLayer](/docs/api-reference/arcgis/deck-layer.md) class, see [pure JS example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/arcgis).
+| Library | Pure JS | React | Overlaid | Interleaved | Docs |
+| ----- | ----- | ----- | ----- | ----- | ----- |
+| [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/latest/) | ✓ | ✓ | | [example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/arcgis) | [link](docs/developer-guide/base-maps/using-with-arcgis) |
```

Comment:
If you add .md to the doc link and start it with `/docs` (leading forward slash), it will work in both GitHub and the website.

Reply:
done